prime-router/metadata/HL7/catchall/hl7/message/ORM_O01.yml

```diff
@@ -38,15 +49,16 @@ resources:
       - .ORC
       - .ORDER_DETAIL.OBR
       - .ORDER_DETAIL.OBSERVATION.NTE
-      - MSH
 
-  - resourceName: Specimen
```

Comment:
Removed specimen here because it wasn't in the mapping inventory and only generates an essentially empty specimen resource because SPM shouldn't be in ORM.